cmd/juju/controller/unregister.go

```diff
@@ -86,7 +85,7 @@ func (c *unregisterCommand) Init(args []string) error {
 	if err := cmd.CheckEmpty(args); err != nil {
 		return err
 	}
-	return nil
+	return c.ConfirmationCommandBase.Init(args)
```

Comment:
Worth taking the time to trace these errors when working with these venerable command modules.

Reply:
Done